src/pudl_archiver/orchestrator.py

```diff
@@ -185,6 +185,12 @@ async def run(self):
         # TODO (daz): pass around changesets instead of persisting them on the instance, this
         # makes the ordering/dependency more explicit
         self._generate_changes(resources)
+
```

Comment:
It'd be nice to pull this logging out of _apply_changes so we can still see "what would have happened if we weren't doing --dry-run":

```suggestion

        logger.info(f"To delete: {self.deletes}")
        logger.info(f"To upload: {self.uploads}")
```

Reply:
Agreed, also it would be nice to have some more clear logging outputs like `No changes detected; no new uploads` or something like that.

Alternatively, we could add documentation here explaining what some of the expected outputs are. I.e.: 

```
To delete: []
To upload: []
```
means that the archive hasn't changed and nothing new will be uploaded.

Reply:
I moved the logs and added a message if no changes are detected